Gaijinizer patch/CommonEvents_596.txt

```diff
@@ -126,19 +127,19 @@ his voice crying for help falls on deaf ears.#Triketos
 文字通り、一滴残らず、干からびるまで
 彼はハーピーモドキたちに『餌場』として扱われるのだ……
 > CONTEXT: CommonEvents_596/232/Dialogue 
-If one Harpy is satisfied, then another one 
-takes its place... and another one...
-They squeeze and suck every drop of semen from
-him till his body is completely dry... #Triketos, moved some of this to the next text box...
+If one Harpy is satisfied, another one takes its place...
+And another one...
+They will suck literally every last drop out of his body,
+until he dries up completely......#Triketos/Strate, moved some of this to the next text box...
 > END STRING
 
 > BEGIN STRING
 『人間』としての彼は……
 
 もはや、どこにもいない……
 > CONTEXT: CommonEvents_596/237/Dialogue 
-In the end, he is treated as nothing more then
-a "feeding ground" for the Harpies.
+In the end, he is treated as nothing more than a
+"feeding ground" for the Harpies...
 
-No person ever saw him again...#Triketos
+As a "human", he does not exist anymore...#Triketos/StrateUnsure
 > END STRING
```

Comment:
Note

Reply:
This could probably be worded like...

"In the end, he no longer exists as a human being, but as
a mere "feeding ground" for the Harpy Spawn."